src/common/Core/Search/SearchFilter.test.ts

```diff
@@ -0,0 +1,17 @@
+
+import { describe, expect, test } from "vitest";
+import { SearchEngineId, searchFilter } from "./SearchFilter";
```

Comment:
Import types only, whereever possible:

```suggestion
import { type SearchEngineId, searchFilter } from "./SearchFilter";
```

Reply:
Fixed and added eslint rule